doc/src/content/atoms/dropdown/DropDownWithNoIcon.md

```diff
@@ -0,0 +1,18 @@
+---
+path: "/dropdown/2"
+component: "DropDown"
+type: "atom"
+title: "DropDown with no icon"
+---
+<codeblock>
+<DropDown
+  options={[
+    { label: 'Apple', value: 'apple' },
+    { label: 'Banana', value: 'banana' },
+    { label: 'Custard Apple', value: 'custard-apple' },
+    { label: 'Dates', value: 'dates' }
+  ]}
+  icon={false}
```

Comment:
The truth and false value are function and boolean respectively which could be changed. May be pass icon prop only if required and don't pass when not required ?